src/api/plugins/video/tests/test_api.py

```diff
@@ -123,3 +123,72 @@ async def test_views_backend_query(http_client: AsyncClient, httpx_mock: HTTPXMo
     }
 
     assert video_views == expected_video_views
+
+
+@pytest.mark.anyio
+async def test_unique_views_backend_query(
+    http_client: AsyncClient, httpx_mock: HTTPXMock
+):
+    """Test the video views endpoint, with parameter unique=True."""
+    # Build 3 video statement fixtures of the same actor
+    view_1_date = "2020-01-01T00:00:00"
```

Comment:
nit: to make it a real timestamp, should it have a timezone/ISO8601 string?
```suggestion
    view_1_date = "2020-01-01T00:00:00.000Z"
```